app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ActiveRecord::Base
+  validates :username, presence: true, uniqueness: true
+  validates :email, presence: true, uniqueness: true
+  has_many :reviews
```

Comment:
this should probably be a has many through relationship, so users have many reviews through products, since users don't have reviews directly.